tests/formatter_tests.py

```diff
@@ -5,10 +5,17 @@
 import pytz
 from chai import Chai
 from dateutil import tz as dateutil_tz
+from dateutil.zoneinfo import get_zonefile_instance
 
 from arrow import formatter
 
 
+def make_full_tz_list():
```

Comment:
Let's import this from `parser_tests.py` so we do not repeat code. Or better yet, we could make a `utils.py` file and import from it in both `formatter_tests.py` and `parser_tests.py`. Either would work, but the second option may be more maintainable.

Reply:
Hi! Ok, done.